src/pages/components/inputs.mdx

```diff
@@ -81,7 +81,7 @@ import IconsCode from '!!raw-loader!../../examples/forms/inputs/MediaInput.tsx';
   <Icons />
 </CodeExample>
 
-### FauxInput
+### Faux input
```

Comment:
@rossmoody just a ping that there are more copy updates needed with the latest merge from `master`.